client/state/data-layer/wpcom/sites/comments/likes/mine/delete/test/index.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import {
+	COMMENTS_UNLIKE,
+	COMMENTS_LIKE,
+	NOTICE_CREATE,
+} from 'state/action-types';
+import {
+	unlikeComment,
+	updateCommentLikes,
+	handleUnlikeFailure,
+} from '../';
+import { local } from 'state/data-layer/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+const SITE_ID = 91750058;
```

Comment:
almost missed this comment... changed on 839e5e1438a366a9236967bc39faf508e48e2cdc.